integrations/integracion-cuenca-catalana/src/cuenca.mapper.ts

```diff
@@ -0,0 +1,16 @@
+import { EmbalseCatalanApi } from './api/cuenca.api-model';
+import { EmbalseUpdateSAIHEntity } from 'db-model';
+import { formatApiDate } from './business';
+
+export function mapApiToEmbalses(
+  apiData: Record<string, EmbalseCatalanApi>
+): EmbalseUpdateSAIHEntity[] {
+  return Object.entries(apiData).map(([id, embalse]) => {
+    return {
+      id: Number(id.replace('-', '')),
+      nombre: embalse.name,
+      aguaActualSAIH: embalse.popup.volume.value, // volumen actual
```

Comment:
i think that is a good idea round the decimals to 2 decimals because now we have this format
10.453.